src/dialogs/more-info/controls/more-info-alarm_control_panel.js

```diff
@@ -29,8 +29,8 @@ class MoreInfoAlarmControlPanel extends LocalizeMixin(PolymerElement) {
           width: 80px;
         }
         .actions mwc-button {
-          min-width: 160px;
-          margin-bottom: 16px;
+          min-width: 135px;
```

Comment:
Do we need to use `min-width`? Can't we use flex?

Reply:
We don't have to use `min-width`. I opted to use the same styles here that the Lovelace card uses:
https://github.com/home-assistant/home-assistant-polymer/blob/fc3f7ca4b224272956ba47d648eda79216b70f39/src/panels/lovelace/cards/hui-alarm-panel-card.ts#L313

It uses `calc(var(--base-unit) * 9)` but it calculates to 135px and this component doesn't have the same potential for variable reuse so I didn't bother using variables with `calc`.

Would it be preferable to use this instead?
```css
flex-basis: 135px;
flex-shrink: 0;
```

Reply:
I was thinking:
```css
flex: 1 0 50%;
```
So it will nicely fill the dialog (assuming we have a max of 2 buttons)

Reply:
That makes the buttons really wide when the dialog is maximized, but I don't think that's a common scenario. I'm good with using flex.

<img width="1395" alt="Screen Shot 2019-10-15 at 10 52 18 AM" src="https://user-images.githubusercontent.com/1522068/66856831-e58ee680-ef3a-11e9-878b-9bfac5c5814d.png">

Reply:
We could give them a `max-width`

Reply:
Done. `max-width: 200px` to match the width of the alarm code input. The two buttons still fill up the dialog when not maximized too.
<img width="914" alt="Screen Shot 2019-10-15 at 11 26 57 AM" src="https://user-images.githubusercontent.com/1522068/66858815-ef1a4d80-ef3e-11e9-932c-832c187c18a7.png">

<img width="904" alt="Screen Shot 2019-10-15 at 11 26 42 AM" src="https://user-images.githubusercontent.com/1522068/66858777-df026e00-ef3e-11e9-9b22-e8815d1ae93f.png">
<img width="476" alt="Screen Shot 2019-10-15 at 11 28 51 AM" src="https://user-images.githubusercontent.com/1522068/66858849-ff322d00-ef3e-11e9-9d2c-7a8b478940da.png">